src/lib.rs

```diff
@@ -128,7 +128,7 @@ impl ZWaveWatcher {
 }
 
 impl manager::NotificationWatcher for ZWaveWatcher {
-    fn on_notification(&self, notification: Notification) {
+    fn on_notification(&self, notification: &Notification) {
```

Comment:
no other change needed \o/ It means well that this should have been like that since the start :)